lldb/source/Interpreter/CommandObject.cpp

```diff
@@ -769,10 +768,10 @@ bool CommandObjectRaw::Execute(const char *args_string,
     handled = InvokeOverrideCallback(argv, result);
   }
   if (!handled) {
-    if (CheckRequirements(result))
-      handled = DoExecute(args_string, result);
+    if (CheckRequirements(result)) {
```

Comment:
I don't think you should remove the `if (!handled)` check at the start of this block; it may be set to true if `InvokeOverrideCallback()` did something.

Reply:
ah my eyes might have been glazing over at this point, I must have misread it. :)